client/src/app/footer/footer.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
I noticed the same empty `ngOnInit` method in the welcome component and removed it together with the `implements OnInit` because we don't use it. Do we need this here?

Reply:
ah it's because its autogenerated by angular cli, i didn't bother to remove it
```
ng g c footer
```